db/migrate/20210910200224_create_imo_delivery_detail.rb

```diff
@@ -0,0 +1,11 @@
+class CreateImoDeliveryDetail < ActiveRecord::Migration[5.2]
+  def change
+    create_table :imo_delivery_details do |t|
```

Comment:
The TwilioSmsDeliveryDetail also has a `delivered_on` attribute. This is because twilio initially returns a state of `queuing` then uses callbacks to tell us when it was delivered. For Imo, the creation date is the delivery date. We could add the field if it makes querying easier. We could also consider adding a `read_on` attribute to both models, but I think the `updated_at` essentially captures that.

Reply:
Lets go with a timestamp right away, something like `read_at` .. that way its very explicit and clear when we want to calculate time between delivery and reading.

Reply:
👍

Reply:
Made a story for this: https://app.shortcut.com/simpledotorg/story/5171/add-read-at-to-twilio-detail